### ISSUE TITLE
Pension Scheme - 01 January 2018

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#342]
The People's Pension https://thepeoplespension.co.uk/
Husky Finance https://huskyfinance.com/
Aviva https://www.aviva.co.uk/business/workplace-pensions/
Smart Pensions https://www.autoenrolment.co.uk/
NEST http://www.nestpensions.org.uk/schemeweb/NestWeb/public/home/contents/homepage.html

**Comment by iteles** [comment]
Why is this a `priority-2` issue?

**Comment by markwilliamfirth** [comment]
@iteles was conducting quick initial research as it popped up on payroll - doesn't need to be fully actioned until end of the year

**Comment by iteles** [comment]
Moving to p4 in line with #349

**Comment by iteles** [comment]
We'll be postponing this for 3 months to allow us to write up all of the information for dwylers in a way that will enable informed decision making. http://ukpensionsautoenrolment.co.uk/survival-guide/postponement/

A letter will go out to explain this.

**Comment by rub1e** [comment]
# Pensions - it's time to adult

## Pensions are fun! (And sensible)

Let's just assume you want to plan for the future and start your pension. Good for you, that's smart.

The way this works is that you put some of your salary into a pot, and your kind employer dwyl matches it (up to 1% of your annual salary). 

Your pension pot is managed by dwyl's pension provider, who invest your money for you until you're old enough to retire and buy an [annuity](https://www.pensionsadvisoryservice.org.uk/about-pensions/retirement-choices/annuities) (or whatever you choose).

In return for managing your money, the pension provider charges a fee, usually in the form of a percentage of your whole pension pot, a percentage of your monthly contributions, a flat monthly fee, or a combination of these. 

### PLEASE DO NOT JUST READ THIS SHORT SUMMARY - IT'S NO SUBSTITUTE FOR ACTUALLY READING AND LEARNING ABOUT PENSIONS PROPERLY. THIS IS YOUR FUTURE!

### INVESTMENTS CAN FALL IN VALUE AS WELL AS RISE: IT'S YOUR RESPONSIBILITY TO UNDERSTAND THE RISKS

## Options

dwyl needs to choose a provider now and has identified two main contenders for your consideration (outlined below, along with a few that were rejected).

In addition to being good value for dwylers financially :money_mouth_face:, key considerations were ease of use (for you and for dwyl) and flexibility (can you tailor how the provider invests on your behalf, more/less risk, ethical investments etc.).

(Not to mention obvious hygiene factors like having an accessible call centre or usable website.)

### The People's Pension (TPP)

(Try not to be swayed by the name alone!)

* Fee: 0.5% of your total pension pot every year (see note below)
* Investment choice, flexibility, & security
  * Ethical and Sharia investment options available
  * Low effort “investor profiles” (e.g. cautious/risky) give employees control without taking up their time, but more granular control is also available
  * Glide Path (see note below)
* UK Call Centre (inc. evenings & weekends)

TPP's parent company is a well-established non-profit and many companies small & large use them as a pension provider. They integrate well with dwyl's accounting system. dwyl would pay a small one-off setup fee of £500.

### Nest

* Fee: 1.8% on all contributions + 0.3% total fund annual fee (see note below, sounds worse than it is!)
* Investment choice, flexibility, & security
  * Ethical and Sharia investment options available
  * Granular control available, but more user input required, doesn't look as off-the-shelf as TPP
  * Glide Path (see note below)
* UK Call Centre (inc. evenings & weekends), live-chat online

Nest is the government workplace pension provider (*N*ational *E*mployment *S*avings *T*rust), so very many companies use them. They also integrate well with dwyl's accounting system. No setup fee.

### Rejected

* **Smart Pensions**: would have made the list if not for being slightly worse in every category. 0.75% fee, Sharia but no ethical fund, have to email in for granular control rather than do online, call centre only during working hours.
* **Legal & General** - rejected because website was dense, corporate, and confusing - the opposite of holding your hand through difficult financial decisions; but basically the same services as above with 0.5% annual fee. No integration.
* **Aviva** - c. £40/m for dwyl, plus 0.75% member annual fee - unclear what the extra costs get you, seems extremely bad value for both dwyl and dwylers. Mixed reports on integration.
* **Now Pensions** - 0.3% member annual fee + £1.50/m (can be paid by employer or employee), apparently very little flexibility, sharia but no ethical fund (though they subscribe to responsible investment principles). Would you believe their site doesn’t use SSL?!
* **Creative Auto-enrollment**: £20/m for dwyl, plus 0.4% member annual fee + £2/m. Literally nothing on the site about ethical/sharia/flexibility etc. And, frankly, the whole site looked really dodgy. Bad value and bad vibes. 

### Notes

#### Difference in fees

TPP and Nest take different approaches, with different results. I only want to make this note because Nest's 1.8% contribution fee looks like a bigger deal than it really is. 

Let's say you make £24k in Year 1. You and dwyl put 1% into the pot - that's £480. 

* Nest takes 1.8% of contributions (£480) + 0.3% of total pot (£480) - so Nest takes £10.08
* TPP takes 0.3% of total pot - so TPP takes £1.44

:boom: :boom: :boom:

No brainer, right? A tenner vs a bus ticket? But now let's say it's the future and you're earning £75k and your pension pot has fared well and grown to £100k! You and dwyl put in 1%, so that's £1,500

* Nest takes 1.8% of £1,500 and 0.3% of 100k = £327
* TPP takes 0.5% of 100k = £500

Not so easy now, is it! 

The point of the above exercise was just to illustrate the importance of understanding this stuff yourselves. Because, and I can't stress this enough,
### THIS IS NOT FINANCIAL ADVICE, DO YOUR OWN DUE DILIGENCE BEFORE EXPRESSING A VIEW

#### Glide path

> The glide path creates an asset allocation that becomes more conservative (i.e., includes more fixed-income assets and fewer equities) the closer a fund gets to the target date.

Or, in English, the provider will automatically start moving your pension pot into more secure, less risky investments as you near retirement age. You don't need to micro-manage your pension if you don't want to.

## In conclusion

Read it, digest it, think about it, ask about it. If you don't have questions about how pensions work, you probably haven't looked into it enough. For example, you'll notice I haven't mentioned the providers' investment track records. That's because

### THIS IS NOT FINANCIAL ADVICE OR A SUBSTITUTE FOR YOUR OWN RESEARCH

**Comment by iteles** [comment]
We will need to check that this piece of info is clear on the letter templates we get from TPP:
http://www.thepensionsregulator.gov.uk/doc-library/increases-in-minimum-contributions-automatic-enrolment.aspx
<img width="855" alt="screen shot 2018-02-21 at 17 05 58" src="https://user-images.githubusercontent.com/4185328/36494410-dd001f56-1729-11e8-8039-bb960dba57a0.png">

dwylers must be set up automatically on TPP and then if they decide to opt out, must create their own account on TPP to opt-out by 31-Mar-2018.

**Comment by rub1e** [comment]
UPDATE: following extensive research, consideration, and debate, @iteles has decided that the best provider for _dwyl's_ needs, and - more importantly - for _dwylers'_ needs, is `The People's Pension` 🎆 

We're halfway through the setup process with TPP, and we expect letters to go out this week or next week with all the details for employees. Watch this space... 👁

**Comment by iteles** [comment]
Letters for this went out earlier this week to @naazy, @SimonLab, @Cleop and @Danwhy.

@rub1e, over to you to finish the enrolment once #415 is complete 👍

**Comment by rub1e** [comment]
We're signed up withe TPP and I've connected the Xero payroll system as well. 

I'm waiting for a couple of queries to be answered, but the next step is for me to upload all dwyl employees'  particulars, and then distribute login details so people can access their accounts on the TPP website.

@iteles and I will also have a chat with our bookkeeper and check that everything looks like it was set up properly (w/c 12/3)

**Comment by rub1e** [comment]
All employee data has now been submitted - so with any luck employees will receive a communication from TPP tomorrow (21/3). (I'll be checking in to make sure)

**Comment by rub1e** [comment]
Update - Xero and TPP aren't integrating *at all*. As usual, each party is blaming the other's systems. 

I've done everything I can to make sure the details on each system are correct - the problem is on one/both of their ends, but neither is taking responsibility.

To make matters worse, we owe TPP a bunch of money but we can't settle up until the systems are talking to each other. They won't chase us until the payment is 90 days overdue (September), but still this is hardly ideal.

**Comment by rub1e** [comment]
Update - Xero have confirmed that the problem is on the TPP side, and I've sent TPP the instructions that Xero sent me. 

I'm waiting for this to be complete before I finish the last two tasks on this list

**Comment by rub1e** [comment]
Just been chased for pensions data per https://github.com/dwyl/hq/issues/474#issuecomment-422417005 (will write process up when I've figured it out!) and it jogged my memory - @iteles did you or @nelsonic ever log in to The People's Pension to claim your refund?

**Comment by iteles** [comment]
Ah, no, I'll add it to my list!

**Comment by rub1e** [comment]
Bump @iteles! 

I've added this to the top of your `project` - it's not urgent, just needs to be done so I can reconcile wages as part of #461

**Comment by iteles** [comment]
As you saw yesterday:
![image](https://user-images.githubusercontent.com/4185328/48549649-498ed200-e8c8-11e8-83fc-b6785a09f9e2.png)


I've sent them a support request, let's see!

**Comment by iteles** [comment]
@rub1e As we discussed, the reply I eventually got was:
> Our records show you opted out of the pension scheme on 31st May 2018 and a refund of your contributions was issued to your employer on 1st July 2018.

Please check for the refund and take the necessary steps 👍
